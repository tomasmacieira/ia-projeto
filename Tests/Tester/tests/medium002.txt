ROW	3	0	4	1	4	1	4	0	2	1
COLUMN	1	2	1	3	0	3	4	4	2	0
3
HINT	0	7	R
HINT	6	6	M
HINT	2	0	C
