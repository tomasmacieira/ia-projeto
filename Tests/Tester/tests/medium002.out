.....lmR..
..........
C..t.c..c.
...m......
.t.b..lr..
.b........
.....lMmr.
..........
......lr..
..c.......
